sdk/src/main/java/com/hedera/hashgraph/sdk/IPAddress.java

```diff
@@ -0,0 +1,49 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.ByteString;
+
+class IPv4Address {
+    IPv4AddressPart network;
+    IPv4AddressPart host;
+
+    IPv4Address() {
+    }
+
+    static IPv4Address fromProtobuf(ByteString address) {
+        return new IPv4Address()
```

Comment:
This class and `IPv4AddressPart` are not necessary. Java already has a class that can parse IPv4 addresses:

`InetAddress.getByAddress(address.toByteArray()).getHostAddress()`